src/assets/styles/SignUpPage.scss

```diff
@@ -0,0 +1,178 @@
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  .sign-up-page__form {
+    margin: auto;
+    width: 100%;
+    height: 404px;
+    border: 1px solid #fff;
+    padding: 40px 30px;
+    border-radius: 5px;
+
+    p {
+      color: $text;
+      text-align: center;
+      font-size: 1.2rem;
+      margin-bottom: 30px;
+    }
+    .sign-up-page__signup-link {
+      display: flex;
+      justify-content: flex-start;
+      font-size: 0.98rem;
+      a {
+        margin-left: 0.4rem;
+        color: $text;
+        &:hover {
+          color: $background;
+        }
+      }
+    }
+    .sign-up-page__social-links {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      a {
+        &:hover {
+          transform: scale(1.3);
+          transition: all 0.2s ease-in-out;
+        }
+      }
+
+      img {
+        height: 30px;
+        padding-left: 1rem;
+      }
+      span {
+        color: white;
+      }
+    }
+  }
+}
+
+.sign-up-page__navigation-bar {
+  width: 80%;
+  margin: 0 auto;
+  position: fixed;
+  top: 30;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+
+  .sign-up-page__logo__img {
+    width: 170px;
+    height: 30px;
+    margin: 0;
+    display: inline-block;
+    padding: 0;
+  }
+
+  span {
+    background-color: $text;
+    padding: 8px 25px;
+    border-radius: 5px;
+    font-size: 1.1rem;
+
+    a {
+      color: $black;
+
+      &:hover {
+        color: $background;
+      }
+    }
+  }
+}
+
+.sign-up-page__heading {
+  color: $text;
+  font-size: 1.1rem;
+  text-align: center;
+}
+
+.sign-up-page__btn {
+  background-color: $background !important;
+  color: $text !important;
+  width: 100% !important;
+  padding: 12px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 0;
+  border-radius: 0.285714rem;
+  cursor: pointer;
+}
+
+.sign-up-page__form-container {
+  width: 385px;
+}
+.sign-up-page__logo {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 15px;
+  p {
+    color: $text;
+    padding: 10px;
+  }
+}
+
+@media screen and (max-width: 400px) {
+  .sign-up-page {
+    .sign-up-page__form {
+      border: none;
+      padding-top: 0;
+
+      .sign-up-page__signup-link {
+        display: none;
+      }
+
+      .sign-up-page__social-links {
+        padding-top: 30px;
+      }
+    }
+
+    .sign-up-page__navigation-bar {
+      .sign-up-page__logo__img {
+        width: 120px;
+        height: 20px;
+      }
+      span {
+        padding: 6px 15px;
+        a {
+          font-size: 12px;
+        }
+      }
+    }
+    .sign-up-page__heading {
+      font-size: 13px;
+      width: 270px;
+      margin: 10px auto;
+    }
+    .sign-up-page__form-container {
+      width: 330px;
+      margin-top: 150px;
+    }
+  }
+
+  .ui .input,
+  .sign-up-page__btn {
+    height: 30px !important;
```

Comment:
This is the first page and you are already creating important tags, this is not a good sign